includes/dart-integrations/index.mdx

```diff
@@ -12,6 +12,6 @@
 | [`Isar`](/platforms/dart/guides/flutter/integrations/isar-instrumentation) |                  |            |      ✓      |                        |
 | [`Logging`](/platforms/dart/guides/flutter/integrations/logging) |                  |     ✓      |             |           ✓            |
 | [`Routing`](/platforms/dart/guides/flutter/integrations/routing-instrumentation) |                  |            |      ✓      |           ✓            |
-| [`Slow and Frozen Frames`](/platforms/dart/guides/flutter/integrations/slow-and-frozen-frames-instrumentation) |  |  |      ✓     |                        |
+| [`Slow and Frozen Frames`](/platforms/dart/guides/flutter/integrations/slow-and-frozen-frames-instrumentation) | ✓ (not if `WidgetsFlutterBinding` is initialized before `SentryFlutter.init`) |  |      ✓     |                        |
```

Comment:
I'm not sure how to handle this, it is only enabled [under certain conditions](https://docs.sentry.io/platforms/dart/guides/flutter/integrations/slow-and-frozen-frames-instrumentation/#early-widgets-binding-initialization)

Reply:
saying it is auto-enabled is not entirely correct

Reply:
I pushed a commit, to make some aesthetic changes, and just put a * next to the ✔️ , then an alert at the bottom of the table, and an alert in the actual integration page, since it wasn't so obvious to me there either. I think it's good to go

Thanks for following up!